### ISSUE TITLE
Aria-live region not working as expected. 

### ISSUE THREAD

**Issue by haritha2112** [nvaccess/nvda#12278]
### Steps to reproduce:

In Chrome or Firefox, visit any one of the following: 
1.  [JSBin](https://output.jsbin.com/lapiras)
2. [Live Region Playground](https://dequeuniversity.com/library/aria/liveregion-playground)

### Expected behavior:
For the JSBin example, the NVDA reads the 'Button 1 Clicked' when Button 1 is clicked, and reads  'Button 2 Clicked' when Button 2 is clicked. 
For the Live region playground, the NVDA reads every new line that is added in the live region

### Actual behavior:
In both the cases, the NVDA does not read the updated text. I do not face this issue with VoiceOver on Chrome and Safari. 

### System configuration
NVDA installed from source: Microsoft Store
NVDA version: 2019.3.1
Windows version: 10.0.19041

#### Name and version of other software in use when reproducing the issue:
Firefox: 87.0
Chrome: 89.0.4389.114

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors
No

#### If add-ons are disabled, is your problem still occurring?
I do not have any add-ons installed

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

**Comment by feerrenrut** [comment]
2019.3.1 is quite an outdated version of NVDA. The most recent release is 2020.4. For technical reasons there are some serious limitations to the Windows store version of NVDA (these are described on the store listing, and include being able to work with Chrome and Firefox). Unless you are using Windows 10 S, or have some other specific requirement to use the Microsoft Store version please use the version available from https://www.nvaccess.org/download/. This is mentioned in the first paragraph of the readme file for this repository: https://github.com/nvaccess/nvda

I can't reproduce this issue testing with NVDA 2020.4.
